keras/layers/convolutional.py

```diff
@@ -1459,46 +1484,79 @@ class ZeroPadding2D(Layer):
 
     # Input shape
         4D tensor with shape:
-        (samples, depth, first_axis_to_pad, second_axis_to_pad)
+        `(samples, channels, rows, cols)` if dim_ordering='th'
+        or 4D tensor with shape:
+        `(samples, rows, cols, channels)` if dim_ordering='tf'.
 
     # Output shape
         4D tensor with shape:
-        (samples, depth, first_padded_axis, second_padded_axis)
+        `(samples, channels, padded_rows, padded_cols)` if dim_ordering='th'
+        or 4D tensor with shape:
+        `(samples, padded_rows, padded_cols, channels)` if dim_ordering='tf'.
     '''
 
-    def __init__(self, padding=(1, 1), dim_ordering='default', **kwargs):
+    def __init__(self,
+                 padding=(1, 1),
+                 top_pad=1,
+                 bottom_pad=1,
+                 left_pad=1,
+                 right_pad=1,
+                 dim_ordering='default',
+                 **kwargs):
+        import sys
```

Comment:
Imports should go at the top of the file. Also this import is not used anywhere.